### PR TITLE
IG-11031: make spark optional for zeppelin

### PR DESCRIPTION
**Pull request by gshatz**
This PR is a prerequisite for https://github.com/iguazio/provazio/pull/508

### CODE REVIEW

File: stable/zeppelin/templates/zeppelin-configmap.yaml [hunk 1]
```diff
@@ -52,5 +52,7 @@ data:
     fi
 
     /docker-zeppelin.sh \
+{{- if .Values.spark }}
             --master-hostname={{ .Values.spark.hostname }} \
```
Comment (bivas) [inline stable/zeppelin/templates/zeppelin-configmap.yaml:56]: I don't think the `docker-zeppelin.sh` script can operate without these params.

**Review by bivas — CHANGES_REQUESTED** [review]
Please test this deployment.

File: stable/zeppelin/templates/zeppelin-configmap.yaml (hunk 1 above)
Comment (gshatz) [reply stable/zeppelin/templates/zeppelin-configmap.yaml:56]: Tested both with spark and without. without spark it logs that it expect spark details but it launches zeppelin without any problems. you can see it here:
https://github.com/kubelytics/kubelytics/blob/c87710605fa86ead89edda9092e3ee05b5d573e2/services/zeppelin/docker/docker-zeppelin.sh

**Comment by gshatz** [comment]
After discussion with Adi it was decided to to make Spark mandatory for Zeppelin deployment. 
Closing this PR

**Pull request closed without merging**